nexus_constructor/component_tree_model.py

```diff
@@ -26,7 +26,7 @@ class ComponentTreeModel(QAbstractItemModel):
     def __init__(self, model: Model, parent=None):
         super().__init__(parent)
         self.model = model
-        self.components = self.model.entry.instrument.component_list
+        self.components = self.model.entry.instrument.get_components()
 
     def columnCount(self, parent: QModelIndex) -> int:
```

Comment:
Can we please remove this function? Why do we need a function to return an integer?